test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -187,43 +254,30 @@ public BlobTokenValidator(string authenticationScheme)
                 AuthenticationScheme = authenticationScheme;
             }
 
-            public string AuthenticationScheme { get; set; }
+            public string AuthenticationScheme { get; }
 
-            public bool CanValidateToken
-            {
-                get
-                {
-                    return true;
-                }
-            }
+            public bool CanValidateToken => true;
 
             public int MaximumTokenSizeInBytes
             {
-                get
-                {
-                    return 2*2*1024;
-                }
-
-                set
-                {
-                    throw new NotImplementedException();
-                }
+                get { return 2*2*1024; }
```

Comment:
Sure. I removed this value and replaced it with a NIE as it's actually not even used.